src/Components/About/About.jsx

```diff
@@ -0,0 +1,49 @@
+// MUI Imports
+import { Grid, Card, Typography } from "@mui/material";
+
+// Component Imports
+import GitHubProfile from "../GitHubProfile/GitHubProfile";
+
+function About() {
+    return (
+        <Grid
+            container
+            direction="column"
+            justifyContent="center"
+            alignItems="center"
+            sx={{
+                marginTop: '4em',
+            }}
+        >
+            <GitHubProfile />
+            <Card
+                elevation={4}
+                sx={{
+                    margin: '2em',
+                    padding: '1em',
```

Comment:
I tend to stay away from using em to size things.  em units are tied to the fontsize being used in the app.  If you decided that you wanted to change the font size at a lower node in the dom tree, it might cascade down to where you're using it and change the size of your grid.  I think using rem is a better option, it's similar to em but not connected to the fontsize.